functional_tests/scylla_operator/test_functional.py

```diff
@@ -741,3 +746,46 @@ def _check_pod_stats(node):
     final_target_pod_stats = _check_pod_stats(target_node)
     assert final_target_pod_stats
     assert final_target_pod_stats.get("restart_count") == origin_target_pod_stats.get("restart_count")
+
+
+@pytest.mark.required_operator("v1.8.0")
+def test_ssl_autogenerated(db_cluster: ScyllaPodCluster, tmp_path: path.Path):
+
+    cluster_name = db_cluster.k8s_cluster.k8s_scylla_cluster_name
+
+    crt_filename = tmp_path / 'tls.crt'
+    key_filename = tmp_path / 'tls.key'
+    ca_filename = tmp_path / 'ca.crt'
+
+    crt_data = db_cluster.k8s_cluster.kubectl(fr"get secret/{cluster_name}-local-client-ca -o jsonpath='{{.data.tls\.crt}}'",
+                                              namespace=db_cluster.namespace)
+    key_data = db_cluster.k8s_cluster.kubectl(fr"get secret/{cluster_name}-local-client-ca -o jsonpath='{{.data.tls\.key}}'",
+                                              namespace=db_cluster.namespace)
+    ca_data = db_cluster.k8s_cluster.kubectl(fr"get secret/{cluster_name}-local-serving-ca -o jsonpath='{{.data.tls\.crt}}'",
+                                             namespace=db_cluster.namespace)
+
+    crt_filename.write_bytes(base64.decodebytes(bytes(crt_data.stdout.strip(), encoding='utf-8')))
+    key_filename.write_bytes(base64.decodebytes(bytes(key_data.stdout.strip(), encoding='utf-8')))
+    ca_filename.write_bytes(base64.decodebytes(bytes(ca_data.stdout.strip(), encoding='utf-8')))
+
+    log.debug(ca_filename)
+    log.debug(ca_filename.read_text())
+    log.debug(crt_filename)
+    log.debug(crt_filename.read_text())
+    log.debug(key_filename)
+    log.debug(key_filename.read_text())
```

Comment:
These look like good candidates for loop to avoid code duplication.

Reply:
add a loop here